### DISCUSSION TITLE
Update for styled-components v.6

### DISCUSSION THREAD

**Discussion by drMonro** [ben-rogerson/twin.macro#819] [General] [answered]
Will there be an update for styled-components v.6?

**Comment by ben-rogerson** [comment, marked as answer]
I've already pushed [a PR for styled-components v6 support](https://github.com/ben-rogerson/twin.macro/pull/818) which will be in the next version of twin - I'll also push updates to all the twin examples so they're running with sc@v6.
If you want support right now, just add `sassyPseudo: true` to your twin config.
